exchange/non_bid_reason.go

```diff
@@ -6,9 +6,9 @@ package exchange
 type NonBidReason int
 
 const (
-	NoBidUnknownError                      NonBidReason = 0   // No Bid - General
+	NoBidUnknownError                      NonBidReason = 0 // No Bid - General
+	ResponseRejectedGeneral                NonBidReason = 300
 	ResponseRejectedCategoryMappingInvalid NonBidReason = 303 // Response Rejected - Category Mapping Invalid
-	ResponseRejectedPrivacy                NonBidReason = 305
```

Comment:
Thanks. I'm working on making 305 a thing, but right now it's not defined in the spec. I expect we can change this to 305 the following release.